pyaedt/application/Design.py

```diff
@@ -3914,3 +3914,27 @@ def set_temporary_directory(self, temp_dir_path):
         """
         self.odesktop.SetTempDirectory(temp_dir_path)
         return True
+
+    @pyaedt_function_handler()
+    def get_design_settings(self):
```

Comment:
@nnetake Why not making it a property such as `design_settings`? Could it create conflicts with other attributes?

Reply:
@nnetake I think @MaxJPRey intended that the method name could be renamed design_settings from get_design_settings

Reply:
okay sure. I will modify the method name. Thanks!

Reply:
> @nnetake I think @MaxJPRey intended that the method name could be renamed design_settings from get_design_settings. I think makes sense

Exactly. Thanks for adding some clarification @maxcapodi78 .